### PR TITLE
feat: read market's bad debt from the API

### PR DESCRIPTION
**Pull request by gleiser-oliveira**
### [evm app](https://github.com/VenusProtocol/venus-protocol-interface/tree/main/apps/evm/)
- Read market's bad debt from the API
- Use the value when calculating the interest and utilization rates

### CODE REVIEW

File: apps/evm/src/clients/api/queries/useGetPool/__snapshots__/index.spec.tsx.snap [hunk 1]
```diff
@@ -5,6 +5,7 @@ exports[`useGetPool > returns the correct asset 1`] = `
   "pool": {
     "assets": [
       {
+        "badDebtMantissa": 0n,
```
Comment (therealemjy) [inline apps/evm/src/clients/api/queries/useGetPool/__snapshots__/index.spec.tsx.snap]: Just to cover this new feature with tests, it would be good to have some test assets with a non-zero bad debt.

File: apps/evm/src/clients/api/queries/useGetPools/getPools/getApiPools/index.ts [hunk 2]
```diff
@@ -80,6 +80,7 @@ export interface ApiMarket {
   lastCalculatedXvsAccruedBlockNumber: string | null;
   supplyCapsMantissa: string;
   borrowCapsMantissa: string;
+  badDebt: string;
```
Comment (therealemjy) [inline apps/evm/src/clients/api/queries/useGetPools/getPools/getApiPools/index.ts]: I understand this is the schema from the API, but I'd recommend updating it at some point to be consistent with the naming convention: `badDebtMantissa`.

Comment (gleiser-oliveira) [reply apps/evm/src/clients/api/queries/useGetPools/getPools/getApiPools/index.ts]: Agreed, let's do it before going live

**Review by therealemjy — APPROVED** [review]
